### PR TITLE
CHAIN-177: add otterscan customization

### PR DESCRIPTION
**Pull request by bholzman**

### CODE REVIEW

File: scripts/envs/demo.yml [hunk 1]
```diff
@@ -57,7 +61,7 @@ services:
     count: 1
     is_essential: "true"
   otterscan:
-    image: otterscan/otterscan
+    image: 851725450525.dkr.ecr.us-east-2.amazonaws.com/otterscan
```
Comment (bflood366) [inline scripts/envs/demo.yml:64]: are there terraform changes needed for this ecr registry.

**Review by bflood366 — APPROVED** [review]
looks great

**Review by imykolenko — APPROVED** [review]
nice, customized block explorer!
otterscan's docker registry was also very slow.

File: scripts/envs/demo.yml (hunk 1 above)
Comment (bholzman) [reply scripts/envs/demo.yml:64]: Indeed there are. I made them, and also refactored the code there a bit to make an `ecr_repo` module.

**Pull request merged**